bigframes/core/blocks.py

```diff
@@ -2484,7 +2484,10 @@ def to_pandas(self) -> pd.Index:
         index_columns = list(self._block.index_columns)
         dtypes = dict(zip(index_columns, self.dtypes))
         expr = self._expr.select_columns(index_columns)
-        results, _ = self.session._execute(expr)
+        results, _ = self.session._execute(
+            expr,
+            sorted=ordered if (ordered is not None) else self.session._strictly_ordered,
```

Comment:
Is it too late to rename this one too?

Reply:
renamed remaining `sorted` to `ordered` in new revision